### PR TITLE
Bowling scoresheet in progress

### PR DESCRIPTION
**Pull request by lroliphant**

### CODE REVIEW

File: spec/gameSpec.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  });
+
+	describe('frames', function() {
+
+		it('consists of 10 frames', function(){
+			expect(game.frameList.length).toEqual(10);
+		});
+
+		it('ends on frame 10/final frame', function() {
+			expect(game.frameList[9].isFinalFrame).toBe(true);
```
Comment (ptolemybarnes) [inline spec/gameSpec.js]: Breaking encapsulation & magic number. You might want a function, #getFrame, that returns a frame at a particular index.

File: public/src/game.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+var Game = function(frame) {
+  this.throwList = [];
```
Comment (ptolemybarnes) [inline public/src/game.js]: Not sure why the Game keeps a complete throwList as it's not needed by the runningTotal

File: public/src/game.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    new Frame(),
+    new Frame(),
+    new Frame(),
+    new Frame(),
+    new Frame(true),
+  ];
+};
+
+Game.prototype.runningTotal = function() {
+  var trackThrow = 0;
```
Comment (ptolemybarnes) [inline public/src/game.js]: trackThrow is having a nice time being added to but I'm not sure if it's a team player.

File: public/src/frame.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
+function Frame(isFinalFrame) {
+  this.throwCount = 0;
+  this.throwScores = [];
+  //if no parameter passed in, isFinalFrame will be set to default of false
+  this.isFinalFrame = isFinalFrame || false;
+}
+
+Frame.prototype.trackThrow = function(pinsDown) {
```
Comment (ptolemybarnes) [inline public/src/frame.js]: Which class is responsible for tracking throws?

File: public/src/frame.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
+function Frame(isFinalFrame) {
```
Comment (ptolemybarnes) [inline public/src/frame.js]: Global variable!

File: public/src/frame.js [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+Frame.prototype.isSpare = function() {
+  return (this.throwScores[0] < 10) && (this.calculateScore() === 10);
+};
+
+Frame.prototype.addBonus = function() {
+  return (this.isSpare()) || (this.isStrike());
+};
+
+Frame.prototype.throwsPerFrame = function() {
+  if(isFinalFrame === true) {
```
Comment (ptolemybarnes) [inline public/src/frame.js]: this is referencing the global variable defined in the spec, so it won't be defined in production :'-(

File: spec/gameSpec.js [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+		it('knows the current frame index', function() {
+			game.throwBall(1);
+			game.throwBall(3);
+			game.throwBall(4);
+			expect(game.currentFrame()).toEqual(game.frameList[1]);
+		});
+
+		it('adds throw scores to array', function() {
+			game.throwBall(1);
```
Comment (ptolemybarnes) [inline spec/gameSpec.js]: Indentation seems to have gone wacky here.

**Comment by ptolemybarnes** [comment]
Hi Lucy, can you change your .travis.yml file to this:

```
language: node_js
node_js:
- "0.12"
before_install:
- "npm install grunt-cli"
- "npm install grunt"
- "npm install grunt-contrib-jasmine"
- "echo \"module.exports=function(grunt){grunt.initConfig({jasmine:{src:'public/**/*.js',options:{specs:'spec/**/*.js',keepRunner:false,vendor:['http://code.jquery.com/jquery-2.1.3.min.js','http://raw.githubusercontent.com/velesin/jasmine-jquery/master/lib/jasmine-jquery.js']}}});grunt.loadNpmTasks('grunt-contrib-jasmine');grunt.registerTask('default',['jasmine']);};\" > Gruntfile.js"
script:
- "grunt"
```
Then CI should be able to run.

**Comment by lroliphant** [comment]
updated travis as requested

**Pull request closed without merging**